roles/servicetelemetry/tasks/component_grafana.yml

```diff
@@ -115,3 +115,74 @@
         state: '{{ "present" if servicetelemetry_vars.graphing.enabled else "absent" }}'
         definition:
           '{{ ds_manifest }}'
+
+    - name: Load Cloud Overview Dashboard
+      k8s:
+        state: '{{ "present" if servicetelemetry_vars.graphing.grafana.dashboards.enabled else "absent" }}'
```

Comment:
TODO: test that disabling dashboards will also remove them (and that I have set the appropriate RBAC rules to allow for that).

Reply:
✅ works as intended